### DISCUSSION TITLE
Support for MikroORM

### DISCUSSION THREAD

**Discussion by brpaz** [aspecto-io/opentelemetry-ext-js#233] [Ideas]
Hello.

I see that a Client was developed for TypeORM, would be nice to have support for [MikroORM](https://mikro-orm.io/), which is a great ORM that seems to be gaining popularity.

They have some similar concepts and ways of initialization. Not sure how much different would be from the TypeORM implementation

**Comment by habmic** [comment]
Hi @brpaz, thanks for reaching out
Would you be interested in making this contribution?
